src/components/one-time-donation/components/Success.tsx

```diff
@@ -0,0 +1,76 @@
+import { Grid, Typography, Button } from '@mui/material'
+import { createStyles, makeStyles } from '@mui/styles'
+import CheckCircleOutlinedIcon from '@mui/icons-material/CheckCircleOutlined'
+import FacebookOutlinedIcon from '@mui/icons-material/FacebookOutlined'
+import InstagramIcon from '@mui/icons-material/Instagram'
+import LinkedInIcon from '@mui/icons-material/LinkedIn'
+import { useTranslation } from 'next-i18next'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    h2: {
+      fontFamily: 'Montserrat',
+      fontStyle: 'normal',
+      fontWeight: '500',
+      fontSize: '32px',
+      lineHeight: '120%',
+      marginTop: '47.25px',
+      marginBottom: '59px',
+      letterSpacing: '-0.5px',
+      color: '#343434',
+    },
+    buttons: {
+      width: '309.55px',
+      height: '70.69px',
+      background: '#32A9FE',
+      border: '0.7441px solid #000000',
+      boxSizing: 'border-box',
+      borderRadius: '35.3275px',
+      margin: 10,
+    },
+  }),
+)
+export default function Success() {
+  const classes = useStyles()
+  const { t } = useTranslation('one-time-donation')
+
+  return (
+    <Grid>
+      <Grid
+        container
+        justifyContent="center"
+        sx={{ '& .MuiSvgIcon-root': { fontSize: 132.5, color: '#4BD12A', marginTop: '58px' } }}>
+        <CheckCircleOutlinedIcon />
+      </Grid>
```

Comment:
You shouldn't be styling the children through a class on the parent. If you need these styles just passed them to the ` <CheckCircleOutlinedIcon />`